src/DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -1099,6 +1099,7 @@ static function bool BingoGoalImpossible(string bingo_event, int start_map, int
         case "LetMeIn":
         case "AlexCloset":
         case "TrophyHunter":
+        case "KnowYourEnemy":
             return start_map > 41 && end_mission <= 4;
```

Comment:
does this mean it's allowed if you start in M04 and end in M05?

Reply:
nvm this seems correct

Reply:
yea I just got this with an NSF HQ start and bingo duration of 2 missions, but mission 5 doesn't have one